coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -84,8 +83,7 @@ public RelToTrinoConverter(HiveMetastoreClient mscClient, Map<String, Boolean> c
    * @return SQL string
    */
   public String convert(RelNode relNode) {
-    RelNode rel = convertRel(relNode, configs);
-    SqlNode sqlNode = convertToSqlNode(rel);
+    SqlNode sqlNode = convertToSqlNode(relNode);
```

Comment:
Why did you ignore the configs?

Reply:
`convertRel(relNode, configs)` is a method from the now deleted `Calcite2TrinoUDFConverter`. I believe the logic of `configs` has been moved to [`CoralToTrinoSqlCallConverter` ](https://github.com/linkedin/coral/pull/491/files#diff-a3a3daba44acc4e0b1208c73b4098f34d49eaecdfec6e29c8afbf289abf00a9aR92)

Reply:
@aastha25 could you verify this change is safe?

Reply:
This should be safe to remove. 
Removing the Rel transformation layer, `Calcite2TrinoUDFConverter`, in Coral IR -> Trino SQL is a huge milestone. Thanks @KevinGe00 for getting us there :)